Makefile

```diff
@@ -6,40 +6,40 @@ all: motioneye/locale/*/LC_MESSAGES/motioneye.mo \
 	msgfmt -f $*.po -o $*.mo
 
 %/motioneye.po: motioneye/locale/motioneye.pot
-	msgmerge --no-wrap -N -U $@ $<
+	# Wrap on 80 characters width to match Weblate
+	msgmerge -w 80 -N -U $@ $<
```

Comment:
I'd be actually fine with `--no-wrap`, if Weblate can be configured to skip it as well?

Reply:
It is even necessary, since `traduki_po.sh` does no line wrap. @jmichault could you adjust this in Weblate, or add me to maintainers, please?